WebRoot/css/zm.css

```diff
@@ -1369,18 +1370,25 @@ INPUT[type="text"].search_input-expanded:focus {
 /* Styles for bubbles in general */
 .addrBubble,
 .addrBubble-selected,
-.addrBubble-actioned,
-.attachmentBubble {
+.addrBubble-actioned {
 	@AddressBubble@
 }
 
+.attachmentBubble {
+	@AttachmentBubble@
+}
+
 .addrBubble:hover,
 .addrBubble-selected,
 .addrBubble-actioned,
 .attachmentBubble:hover {
 	@AddressBubbleActive@
 }
 
+.attBubbleHolder .attachmentBubble:hover .AttachmentClose{
+	@AttachmentBubbleActive@
```

Comment:
in address bubble we are using AddressBubbleOp to indicate operation, so better to go with similar name here AttachmentBubbleOp